crates/stc_ts_file_analyzer/src/analyzer/expr/bin.rs

```diff
@@ -132,10 +132,33 @@ impl Analyzer<'_, '_> {
             .with_child(ScopeKind::Flow, true_facts_for_rhs.clone(), |child: &mut Analyzer| -> VResult<_> {
                 child.ctx.should_store_truthy_for_access = false;
 
+                let mut type_param = vec![];
+                if let TsInstantiation(RTsInstantiation {
+                    type_args: box RTsTypeParamInstantiation { params, .. },
+                    ..
+                }) = &**right
+                {
+                    for param in params {
+                        let ty = child.validate(&**param, ());
+                        if let Ok(ty) = ty {
+                            type_param.push(ty);
+                        }
+                    }
+                };
+                if type_param.is_empty() {
+                    type_param.push(Type::Keyword(KeywordType {
```

Comment:
Please use `Type::any(span, Default::default())` for consistency

Reply:
thx, I forgot this function